.github/pull_request_template.md

```diff
@@ -1,3 +1,23 @@
+To ensure full and complete PR descriptions, please use the following template:
+
+Template based on [this article](https://www.pullrequest.com/blog/writing-a-great-pull-request-description/)
+
+## What?
+Explain what the change is linking any relevant JIRAs or Issues.
+
+## Why?
+Consider what business or engineering goal does this PR achieves.
+
+## How?
+Describe how the change is implemented. Any noteable new libaries, APIs, or features.
+
+## Testing
+Did you add any tests for the change?
+
+
```

Comment:
Minor, as it does not effect how the template is rendered, but there is an extra new-line here compared to the spacing of the other sections.